server/src/main/java/org/opensearch/gateway/remote/model/RemoteIndexMetadataBlobStore.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.gateway.remote.model;
+
+import org.opensearch.cluster.metadata.IndexMetadata;
+import org.opensearch.index.translog.transfer.BlobStoreTransferService;
+import org.opensearch.repositories.blobstore.BlobStoreRepository;
+import org.opensearch.threadpool.ThreadPool;
+
+/**
+ * An implementation of {@link RemoteObjectStore} which is used to upload/download {@link IndexMetadata} to/from blob store
+ */
+public class RemoteIndexMetadataBlobStore extends AbstractRemoteBlobStore<IndexMetadata, RemoteIndexMetadata> {
+    public RemoteIndexMetadataBlobStore(BlobStoreTransferService blobStoreTransferService, BlobStoreRepository blobStoreRepository, String clusterName,
+        ThreadPool threadPool) {
+        super(blobStoreTransferService, blobStoreRepository, clusterName, threadPool);
+    }
+
+}
```

Comment:
Is this the concrete implementation? If yes then you don't need a class, you need an instance
```
AbstractRemoteBlobStore<IndexMetadata, RemoteIndexMetadata> obj
            = new AbstractRemoteBlobStore<>(blobStoreTransferService, blobStoreRepository, clusterName, threadPool);
```

Reply:
I will create instances instead of class.